taskcat/_cli_modules/test.py

```diff
@@ -118,7 +118,7 @@ def run(
             for test in buckets.values():
                 for bucket in test.values():
                     if bucket.name not in deleted:
-                        bucket.delete(delete_objects=True)
+                        bucket.empty()
```

Comment:
Please pay special attention to this. Is this what we want to do?

Reply:
yeah this way we can keep using the same bucket sometimes like:

_account_id=hash(boto3.client('sts').get_caller_identity().get('Account'))
`taskcat-region-_account_id`

Reply:
looked close at the PR 
return f"{prefix}-{hashed_account_id}-{region_obj.name}"
^ works perfect